modules/services/scrutiny-collector.nix

```
@@ -0,0 +1,127 @@
+{ config, osConfig, lib, pkgs, ... }:
+
+let
+  inherit (lib) mkEnableOption mkIf mkOption mkPackageOption types mdDoc;
+
+  cfg = config.services.scrutiny-collector;
+in {
+  meta.maintainers = [ lib.maintainers.dudeofawesome or "dudeofawesome" ];
+
+  options.services.scrutiny-collector = {
+    enable = mkEnableOption (mdDoc "scrutiny-collector");
+
+    package = mkPackageOption pkgs "scrutiny-collector" { };
+
+    config-path = mkOption {
```

Options should be camel cased

```suggestion
    configPath = mkOption {
```